Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/Coupled/Chiller.mo

```diff
@@ -0,0 +1,284 @@
+          extent={{-56,-52},{58,-70}},
+          lineColor={0,0,0},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-103,64},{98,54}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-2,54},{98,64}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={255,0,0},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-101,-56},{100,-66}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,255},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-100,-66},{0,-56}},
+          lineColor={0,0,127},
+          pattern=LinePattern.None,
+          fillColor={0,0,127},
+          fillPattern=FillPattern.Solid),
+        Polygon(
+          points={{-42,0},{-52,-12},{-32,-12},{-42,0}},
+          lineColor={0,0,0},
+          smooth=Smooth.None,
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Polygon(
+          points={{-42,0},{-52,10},{-32,10},{-42,0}},
+          lineColor={0,0,0},
+          smooth=Smooth.None,
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-44,50},{-40,10}},
+          lineColor={0,0,0},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{-44,-12},{-40,-52}},
+          lineColor={0,0,0},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{38,50},{42,-52}},
+          lineColor={0,0,0},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Ellipse(
+          extent={{18,22},{62,-20}},
+          lineColor={0,0,0},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Polygon(
+          points={{40,22},{22,-10},{58,-10},{40,22}},
+          lineColor={0,0,0},
+          smooth=Smooth.None,
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+        Line(points={{62,0},{80,0}},                  color={0,0,255}),
+        Line(points={{80,30},{100,30}},               color={0,0,255}),
+        Line(points={{80,0},{80,30}},                 color={0,0,255}),
+        Line(points={{80,-30},{100,-30}},             color={0,0,255}),
+        Line(points={{80,-30},{80,0}},                color={0,0,255})}),
+        defaultComponentName="chi",
+    Documentation(info="<html>
+<p>This is a model of a squirrel cage induction motor coupled chiller with ideal speed control. The model has electrical interfaces and can be used for simulating microgrids and discussing grid interactions.</p>
+</html>",
+      revisions="<html>
+<ul>
+<li>October 15, 2021,
```

Comment:
see the revision note format.

Reply:
Thank you for viewing, I've corrected the formatting of the release notes and other sections to make sure they match the other models in MBL.